Makefile

```diff
@@ -44,21 +44,25 @@ lint:
 ########################################
 ### Protobuf
 
-proto-gen: proto-update-deps
+proto-gen:
 	@echo "Generating *.pb.go files from *.proto files"
 	$(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace tendermintdev/sdk-proto-gen:v0.2 sh ./scripts/protocgen.sh
```

Comment:
I'm not found `CURDIR` variable.
Is it missing? (or if it was intended. 😄 )

Reply:
I've been curious about it for a long time. HaHa..

Reply:
I think it is `CURDIR="$(pwd)"`

Reply:
CURDIR is one of GNU Make keywords. 

https://stackoverflow.com/questions/52437728/bash-what-is-the-difference-between-pwd-and-curdir

Reply:
Thank you!